plugins/org.eclipse.emf.ecoretools.ale.ide.ui/src/org/eclipse/emf/ecoretools/ale/ide/ui/launchconfig/UiUtils.java

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
```

Comment:
This class looks useful, maybe consider having a copy in GEMOC commons too

Reply:
In the [org.eclipse.gemoc.commons.eclipse.ui](https://github.com/eclipse/gemoc-studio/tree/f413e861a80678eb9d4f55268d032f9c31f90f6a/commons/plugins/org.eclipse.gemoc.commons.eclipse.ui) bundle?